rmf_fleet_adapter/schemas/event_description__charge_battery.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
I don't think we necessarily need this `event_description` schema since we don't support any `ChargeBattery` events in `rmf_task`.
With this description, users can theoretically submit a composable task with a ChargeBattery event injected between other activities but the fleet adapter will not find the right builder from `rmf_task_sequence` to generate the `Model` and `Description` for this event.

Not a blocker imo but it would be good to test what happens when such a compose task is submitted to make sure any errors are gracefully handled.